policy/modules/apps/evolution.te

```diff
@@ -346,6 +368,10 @@ optional_policy(`
 	gnome_stream_connect_gconf(evolution_alarm_t)
 ')
 
+optional_policy(`
+	wm_send_fd(evolution_alarm_t)
```

Comment:
Do you know what this fd is used for?

Also, with the added `wm` call on line 335 (new line num), the `wm` module is already unconditionally required. Please make both lines uncondtional or optional.

Reply:
Is there an error message in the regular log (not audit messages)? If not, how does the code functionally fail?